{
    "type": "bundle",
    "id": "bundle--12b6c9d5-cd0b-44c9-b7a5-ce9b533f809f",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--41c06ea5-9dd6-4559-9287-08b101d58cf9",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-08-22T19:12:33.322266Z",
            "modified": "2023-08-22T19:12:33.322266Z",
            "name": "CVE-2022-25024",
            "description": "The json2xml package through 3.12.0 for Python allows an error in typecode decoding enabling a remote attack that can lead to an exception, causing a denial of service.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-25024"
                }
            ]
        }
    ]
}